base/sim/FairMCApplication.h

```diff
@@ -115,6 +115,8 @@ class FairMCApplication : public TVirtualMCApplication
     TTask* GetListOfTasks();
     FairGenericStack* GetStack();
     TChain* GetChain();
+    /** Register output */
+    void RegisterOutput();
```

Comment:
If this is only needed internally, could you consider moving it into the `private:` section?